### ISSUE TITLE
Elements inside popup not accessible after closing the popup

### ISSUE THREAD

**Issue by carlotrimarchi** [noelboss/featherlight#300]
What kind of issue is this? (put 'x' between the square brackets)

 - [ ] Question. This issue tracker is not the place for questions. If you want to ask how to do
       something, or to understand why something isn't working the way you expect it to, use
       http://stackoverflow.com/questions/ask .
       Provide working code, starting from http://jsfiddle.net/JNsu6/15/.
       We monitor the tag `featherlight.js`.

 - [x] Bug report. If you’ve found a bug, you must provide a minimal example in a CodePen,
       starting from http://jsfiddle.net/JNsu6/15/ .

 - [ ] Feature Request. Make sure there's no good way to do what you want first;
       consider asking on http://stackoverflow.com/questions/ask first.

Here's a working example: http://jsfiddle.net/JNsu6/410/

Basically, I have some radio inputs inside the popup. If you click the button you get the value from the selected radio input.
This works only before opening the popup. After opening the popup, clicking the button will  result in undefined.

**Comment by marcandre** [comment]
Right, `persist=true` removes and keep the elements detached. It has been suggested before to put them back where they were originally.

**Comment by marcandre** [comment]
There's a zillion ways to deal with that, btw:
* don't use `persist`
* keep a reference, taken before the first open
* do the get value in a call back (`beforeClose`)

**Comment by carlotrimarchi** [comment]
I tried not using `persist` but doesn't fix the issue. Besides, if I change radio, without persist the change is lost when opening it again.

The other two points do the trick, but are not really a solution: if I have more inputs it can get annoying storing all the values or keeping a reference when I could access the input directly

**Comment by marcandre** [comment]
Agreed, which is why this issue is still open. Other solution is:
* wait till I fix it

Best solution is  of course:
* write a PR to reinsert the content where it was after closing :-)

**Comment by carlotrimarchi** [comment]
Ahah, you're right!

And sorry, I didn't want to sound demanding. I originally thought that I was missing something out and was doing it wrong
